### DISCUSSION TITLE
Third-Party Mobile Application

### DISCUSSION THREAD

**Discussion by meichthys** [TriliumNext/Trilium#2990] [General]
The mobile site of Trilium is usable but very limited on functionality. Since we have the [ETAPI ](https://github.com/zadam/trilium/wiki/ETAPI) and [Script API](https://github.com/zadam/trilium/wiki/Script-API), it seems like it should be possible for someone to build a third-party mobile application that could sync with the backend server.

@zadam At this point, do you think this would be feasible for someone to implement, or do you think the APIs are still too much in flux?

**Comment by zadam** [comment]
ETAPI is stable (at least in the sense it's not going to radically change) and could be used for this purpose.

However using ETAPI would mean you need to be connected to internet / your server to use the app. The performance probably wouldn't be great as a result.

**Reply by meichthys** [reply]
Thanks. Perhaps the mobile app could cache changes locally until an internet connection is available to sync up. Might be tricky since it wouldn't be a standalone instance, but seems like it should be possible.
